plugin/rfactory.vim

```diff
@@ -1,8 +1,18 @@
 let rfactory_fg_methods = ['create', 'build', 'build_stubbed', 'attributes_for']
-let method_pattern = '\<\%('. join(rfactory_fg_methods, '\|') . '\)' " Non capturing 'or'
+let word_boundry = '\<'
+let method_pattern = '\%('. join(rfactory_fg_methods, '\|') . '\)' " Non capturing 'or'
```

Comment:
This comment applies to a lot of lines, not just this one. How about putting it on its own line?

If I understand correctly:

```
" \%() is a non-capturing group, versus (), which does capture.
```

I assume `or` should be `group` in the comment. Maybe I misunderstood!